packages/communication-react/review/stable/communication-react.api.md

```diff
@@ -1670,6 +1670,7 @@ export type CommonCallControlOptions = {
     cameraButton?: boolean;
     endCallButton?: boolean | /* @conditional-compile-remove(end-call-options) */ {
         hangUpForEveryone?: false | 'endCallOptions';
+        disableEndCallModal?: boolean;
```

Comment:
I feel it is more clear to add confirm or confirmation to the name? Cause end call modal could be modal for some other purposes other than confirmation

Reply:
+1. `Modal` is the UI container instead of the function. Perhaps `disableEndCallConfirmation`/`suppressEndCallConfirmation`/`endCallWithoutConfirmation`